src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -330,8 +332,41 @@ public abstract class GenerateProtoTask extends DefaultTask {
   @Internal("Input captured by getAlternativePaths(), this is used to query alternative path by locator name.")
   abstract MapProperty<String, FileCollection> getLocatorToAlternativePathsMapping()
 
-  @InputFiles
-  @PathSensitive(PathSensitivity.NONE)
+  @Input
+  @Optional
+  Provider<String> getProtocArtifact() {
```

Comment:
this returns a single string as an input but the previous `alternativePaths` property could return a collection of files. do you know what the case is with multiple `alternativePaths`?